package.json

```diff
@@ -2,13 +2,30 @@
   "name": "leaflet.fullscreen",
   "version": "4.0.0",
   "description": "Simple plugin for Leaflet that adds fullscreen button to your maps.",
-  "main": "Control.FullScreen.js",
+  "type": "module",
+  "main": "dist/Control.FullScreen.js",
+  "module": "dist/Control.FullScreen.js",
+  "browser": "dist/Control.FullScreen.umd.js",
+  "exports": {
+    ".": {
+      "import": "./dist/Control.FullScreen.js",
+      "require": "./dist/Control.FullScreen.umd.js"
+    },
+    "./dist/Control.FullScreen.css": "./dist/Control.FullScreen.css"
+  },
+  "files": [
+    "dist/"
```

Comment:
Please add "CHANGELOG.md" so that we have all required documentation on npmjs too